azerothcore/azerothcore-wotlk
# fix(DB/Quest): completable flag of "Gaining Acceptance"

## Changes Proposed:
-  Change flag repeatable for this quest which should be repeatable.
-  `UPDATE `quest_template_addon` SET `SpecialFlags` = 1 WHERE (`ID` = 13662);`

## Issues Addressed:

- Closes #7061
- Closes https://github.com/chromiecraft/chromiecraft/issues/1235

## SOURCE:

https://classic.wowhead.com/quest=7737/gaining-acceptance
https://wowpedia.fandom.com/wiki/Gaining_Acceptance

## Tests Performed:

- No tests performed, simple SQL 0 to 1.
- 


## How to Test the Changes:


1. login and pickup quest "Gaining Acceptance" with thorium brotherhood
2.hand it in and check if it is repeatable (may require pre-requisite rep)
3.

## Known Issues and TODO List:


- [ no known issues]


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
